index.html

```diff
@@ -1334,6 +1336,8 @@ <h3 id="conditional-request-handler">Conditional Request Handler</h3>
 });
 </code></pre>
 
+The above example sets <code>res.etag</code> which is compared against the request's headers by <code>restify.conditionalRequest()</code> to determine if a <code>304</code> should be returned. It also sets the <code>Etag</code> header in the response, so that the client can include it in future requests. Finally, it sets the <code>Last Modified</code> header so <code>restify.conditionalRequest()</code> can determine if the client's copy of the request needs to be updated.
```

Comment:
Heh, I actually follow the same rule. I failed to run `gq}` before closing vim, embarrassing!